From 7b57c376c77d91f208c8c12349c5d406cbdc6c84 Mon Sep 17 00:00:00 2001
From: Rakina Zata Amni <rakina@chromium.org>
Date: Mon, 17 Jun 2024 23:55:18 +0000
Subject: [PATCH] Enable RenderDocument subframe by default

The subframe level of RenderDocument has been approved for 100% launch.
This CL makes the default level for RenderDocument be the "subframe"
level, as the RenderDocument feature itself has already been enabled
by default, but only for the "crashed frame" level.

Bug: 936696
Change-Id: I4c5c976f6f966acfff3ca96eb3eed24aaedcc6b9
Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/5636232
Reviewed-by: Alex Moshchuk <alexmos@chromium.org>
Commit-Queue: Rakina Zata Amni <rakina@chromium.org>
Cr-Commit-Position: refs/heads/main@{#1316172}
---

--- a/content/common/content_navigation_policy.cc
+++ b/content/common/content_navigation_policy.cc
@@ -114,7 +114,7 @@
         {RenderDocumentLevel::kAllFrames, "all-frames"}};
 const base::FeatureParam<RenderDocumentLevel> render_document_level{
     &features::kRenderDocument, kRenderDocumentLevelParameterName,
-    RenderDocumentLevel::kCrashedFrame, &render_document_levels};
+    RenderDocumentLevel::kSubframe, &render_document_levels};
 
 RenderDocumentLevel GetRenderDocumentLevel() {
   if (base::FeatureList::IsEnabled(features::kRenderDocument))
